### PR TITLE
The link for "Cancel and Undo Changes" leads the user to the post page

### PR DESCRIPTION
**Pull request by lucasisaza**
Instead of leading the user back to the index page.

### CODE REVIEW

File: BlogTemplate/Pages/Edit.cshtml [hunk 1]
```diff
@@ -38,7 +38,7 @@
                 <input type="submit" name="publish" id="postsave" class="btn btn-primary btn-sm" value="Publish" asp-page-handler="Publish" />
             </div>
             <div class="col-md-12">
-                <p class="cancel-link"><a href="/Index"><span class="glyphicon glyphicon-triangle-left"></span> Cancel and Undo Changes</a></p>
+                <p class="cancel-link"><a href ="@Model.PathToPost"><span class="glyphicon glyphicon-triangle-left"></span> Cancel and Undo Changes</a></p>
```
Comment (jimmylewis) [inline BlogTemplate/Pages/Edit.cshtml]: 1. the space between the attribute name (href) and the = is invalid.
2. Should this be using a tag helper to generate the link?

Comment (umaslakshmi) [reply BlogTemplate/Pages/Edit.cshtml]: @lucasisaza Look at the changes in #153 in Drafts.cshtml for a good example of using tag helpers to generate the link!

File: BlogTemplate/Pages/Edit.cshtml.cs [hunk 2]
```diff
@@ -24,6 +24,7 @@ public EditModel(BlogDataStore dataStore, SlugGenerator slugGenerator, ExcerptGe
         }
 
         [BindProperty]
+        public string PathToPost { get; set; }
```
Comment (jimmylewis) [inline BlogTemplate/Pages/Edit.cshtml.cs]: This property should be removed.

File: BlogTemplate/Pages/Edit.cshtml.cs [hunk 3]
```diff
@@ -92,6 +96,8 @@ public class EditedPostModel
             [Required]
             public string Body { get; set; }
             public string Excerpt { get; set; }
+            [Required]
+            public int Id { get; set; }
```
Comment (jimmylewis) [inline BlogTemplate/Pages/Edit.cshtml.cs]: You don't want this on the ViewModel.  You're already getting it from the route when you need it.  Instead, you'll want to pass this to the view in a different way (maybe in TempData or ViewData).

**Pull request merged**